pf4j/src/main/java/org/pf4j/processor/ExtensionAnnotationProcessor.java

```diff
@@ -229,4 +254,26 @@ private ExtensionStorage createStorage() {
         return storage;
     }
 
+    private static AnnotationMirror getAnnotationMirror(TypeElement typeElement, Class<?> clazz) {
```

Comment:
Move this method in https://github.com/pf4j/pf4j/blob/master/pf4j/src/main/java/org/pf4j/util/ClassUtils.java